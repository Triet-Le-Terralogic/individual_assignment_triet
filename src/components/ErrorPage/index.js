import React from "react";
import { Link } from "react-router-dom";

export default function ErrorPage() {
  return (
    <div className="Error-page">
      <h1 className="Error-page__header">
        <span>404</span> Error
      </h1>
      <div className="Error-page__body">
        <p>Page not found</p>
        <p>We are sorry but the page you are looking for does not exist.</p>
        <Link to="/">Home page</Link>
      </div>
    </div>
  );
}
