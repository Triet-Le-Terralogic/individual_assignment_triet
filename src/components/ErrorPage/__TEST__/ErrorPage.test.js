import React from "react";
import ErrorPage from "../index";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

describe("<ErrorPage/>", () => {
  const wrapper = shallow(<ErrorPage />);
  it("should render without crash", () => {
    shallow(<ErrorPage />);
  });

  it("should render correctly", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
