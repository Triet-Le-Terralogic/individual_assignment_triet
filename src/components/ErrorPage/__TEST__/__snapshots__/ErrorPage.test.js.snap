// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorPage/> should render correctly 1`] = `
<div
  className="Error-page"
>
  <h1
    className="Error-page__header"
  >
    <span>
      404
    </span>
     Error
  </h1>
  <div
    className="Error-page__body"
  >
    <p>
      Page not found
    </p>
    <p>
      We are sorry but the page you are looking for does not exist.
    </p>
    <Link
      to="/"
    >
      Home page
    </Link>
  </div>
</div>
`;
